src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/domain/sortFavGridItems.ts

```diff
@@ -0,0 +1,22 @@
+import { FavAction } from '../../..//domain/actions/FavAction';
+import { FavoritesDataSource } from '../../../infrastructure/favorites/FavoritesDataSource';
+import { GridItemData } from '../types/GridItemData';
+
+export const sortFavGridItems: (a: GridItemData, b: GridItemData) => number = function (a, b) {
+  const aIsFav = FavoritesDataSource.exists(FavAction.buildFavorite(a));
+  const bIsFav = FavoritesDataSource.exists(FavAction.buildFavorite(b));
+
+  if (aIsFav && bIsFav) {
+    return a.label.localeCompare(b.label);
+  }
+
+  if (bIsFav) {
+    return +1;
+  }
+
+  if (aIsFav) {
+    return -1;
+  }
+
+  return 0;
+};
```

Comment:
Shouldn't we also order the ones that don't have a favourite consistently?

```suggestion
  if (bIsFav && !aIsFav) {
    return +1;
  }

  if (aIsFav && !bIsFav) {
    return -1;
  }

  return a.label.localeCompare(b.label);
};
```

Reply:
Actually no, because we want the original order to still apply.
For instance, label values are already sorted by total consumption, we're just bringing the favorites on top, we don't want to change the original order.